scripts/suite.py

```diff
@@ -172,6 +172,11 @@
                               in the output of teuthology-suite command. -1
                               for a random seed [default: -1].
  --force-priority             Skip the priority check.
+ --disable-num-jobs-check     Skip the number of jobs check. By default,
+                              teuthology will not allow you to schedule more
+                              than JOBS_TO_SCHEDULE_THRESHOLD=500 jobs, because
```

Comment:
you can pass the threshold via format name, like: `more than {jobs_threashold) jobs`,
giving it to format:
```
from teuthology.suite.run import Run
.
.
.
""".format(
    default_machine_type=config.default_machine_type,
    default_results_timeout=config.results_timeout,
    default_ceph_repo=defaults('--ceph-repo',
                            config.get_ceph_git_url()),
    default_suite_repo=defaults('--suite-repo',
                            config.get_ceph_qa_suite_git_url()),
    default_ceph_branch=defaults('--ceph-branch', 'master'),
    jobs_threshold=Run.JOBS_TO_SCHEDULE_THRESHOLD,
)

```